packages/logs/src/domain/logger.spec.ts

```diff
@@ -26,6 +26,19 @@ describe('Logger', () => {
       expect(getLoggedMessage(0).status).toEqual(StatusType.info)
     })
 
+    it("'logger.log' should set 'logger' origin", () => {
+      updateExperimentalFeatures(['forward-logs'])
+      logger.log('message')
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏add a test to make sure `origin` can be overriden by the customer, like:

```ts
logger.log('message', { origin: 'foo' })
expect(getLoggedMessage(0).origin).toEqual('foo')
```

Reply:
Done :)